access/resource_permissions.go

```diff
@@ -264,7 +265,30 @@ func resourcePermissionsRead(d *schema.ResourceData, m interface{}) error {
 
 func resourcePermissionsDelete(d *schema.ResourceData, m interface{}) error {
 	id := d.Id()
+	objectType := d.Get("object_type").(string)
+
+	if objectType == "job" {
```

Comment:
missing resource unit test to cover this code - see `TestResourceJobDelete` for reference.

Reply:
added tests still miss coverage of this added code. `make coverage` should show the coverage locally.